src/app/Http/Controllers/RegisterController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+
+class RegisterController extends Controller
+{
+    public function register(Request $request) {
+        $incomingFields = $request->validate([
```

Comment:
Please move this to a Form Request class.

Reply:
Created a commit for this comment, thank you Sir!